docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -1,145 +1,161 @@
+    `kubectl get pods -n runai | grep -E 'runai-agent|cluster-sync|assets-sync'`
 
-* Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is an essential first step to identify any error messages or connection issues.
+2. Verify Run:ai services logs. Inspecting the logs of the Run:ai services that communicate with the CP is an essential first step to identify any error messages or connection issues.
 
-  Run the following command on each one of the services:
+    Run the following command on each one of the services:
 
-  ```
-  kubectl logs runai-agent -n runai
-  kubectl logs cluster-sync -n runai
-  kubectl logs assets-sync -n runai
-  ```
+    ```bash
+    kubectl logs runai-agent -n runai
+    kubectl logs cluster-sync -n runai
+    kubectl logs assets-sync -n runai
+    ```
 
-* Check the network connection from the runai namespace in your cluster to the Control Plane. You can do that by running a connectivity check pod.
+3. Check the network connection from the `runai` namespace in your cluster to the Control Plane. You can do that by running a connectivity check pod.
 
-  Create a pod within the runai namespace. This pod can be a simple container with basic network troubleshooting tools, such as `curl` or `get`. Use the following command to determine if the pod can establish connections to the necessary Control Plane endpoints:
+    Create a pod within the `runai` namespace. This pod can be a simple container with basic network troubleshooting tools, such as `curl` or `get`. Use the following command to determine if the pod can establish connections to the necessary Control Plane endpoints:
 
-  `kubectl run control-plane-connectivity-check -n runai --image=wbitt/network-multitool --command -- /bin/sh -c 'curl -sSf <control-plane-endpoint> > /dev/null && echo "Connection Successful" || echo "Failed connecting to the Control Plane"'`
+    ```bash
+    kubectl run control-plane-connectivity-check -n runai --image=wbitt/network-multitool --command -- /bin/sh -c 'curl -sSf <control-plane-endpoint> > /dev/null && echo "Connection Successful" || echo "Failed connecting to the Control Plane"'
+    ```
 
-  Replace `control-plane-endpoint` with the URL of the Control Plane in your environment.
+    Replace `control-plane-endpoint` with the URL of the Control Plane in your environment.
 
-* Check potential network issues. Use the following guidelines:
+4. Check potential network issues. Use the following guidelines:
   
     * Ensure that the network policies in your Kubernetes cluster allow communication between the Control Plane and the Run:ai services that communicate with the Control Plane.
-   
     * Check both Kubernetes Network Policies and any network-related configurations at the infrastructure level.
-  
-    * Verify that the required ports and protocols are not blocked. 
+    * Verify that the required ports and protocols are not blocked.
 
 * Contact Run:ai Support
```

Comment:
@jasonnovichRunAI why isn't this a part of the procedure?

Reply:
@jasonnovichRunAI not sure I got it. Currently, it looks weird because it is a bullet after the procedure...